internal/provider/kubernetes/config/envoy-gateway/service.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: envoy-gateway
+  labels:
+    control-plane: envoy-gateway
+spec:
+  ports:
+    - name: grpc
+      port: 18000
+      targetPort: 18000
```

```suggestion
      targetPort: 8001
```

xdsserver default port is 8001?

---

yeah it was 8001, got moved to 18000
things are moving a little fast rn :)
https://github.com/envoyproxy/gateway/blob/a4773ec06b675cc4a6ee1220d0c8ece09053471c/internal/xds/server/runner/runner.go#L27